eclair-core/src/main/scala/fr/acinq/eclair/payment/relay/PostRestartHtlcCleaner.scala

```diff
@@ -329,9 +331,25 @@ object PostRestartHtlcCleaner {
   private def isPendingUpstream(channelId: ByteVector32, htlcId: Long, htlcsIn: Seq[IncomingHtlc]): Boolean =
     htlcsIn.exists(htlc => htlc.add.channelId == channelId && htlc.add.id == htlcId)
 
+  def groupByOrigin(htlcsOut: Seq[(Origin, ByteVector32, Long)], htlcsIn: Seq[IncomingHtlc]): Map[Origin, Set[ChannelIdAndHtlcId]] =
+    htlcsOut
+      .groupBy { case (origin, _, _) => origin }
+      .view
```

Comment:
Added a view here since `mapValues` without it was marked as deprecated.

Reply:
Please revert, this is done in order to keep compatibility with eclair-mobile / phoenix who are on older akka versions.
We will fix these deprecation warnings once phoenix / eclair-mobile don't depend on eclair-core anymore.